core/domain/state_domain.py

```diff
@@ -188,6 +210,22 @@ def validate(self):
         """Validates all properties of Hint."""
         self.hint_content.validate()
 
+    @classmethod
+    def convert_html_in_hint(cls, hint_dict, conversion_fn):
+        """Checks for HTML fields in the hints and converts it
+        according to the conversion function.
+
+        Args:
+            hint_dict: dict. The hints dict.
+            conversion_fn: function. The function to be used for converting the
+                HTML.
+        Returns:
+            hint_dict: dict. The converted hints dict.
```

Comment:
Remove hint_dict. Returns should only have the return type, check existing docstrings for reference

Reply:
done